app/jobs/risc_delivery_job.rb

```diff
@@ -10,11 +10,11 @@ class RiscDeliveryJob < ApplicationJob
 
   retry_on(
     *NETWORK_ERRORS,
-    wait: :exponentially_longer,
+    wait: :polynomially_longer,
     attempts: 2,
   )
   retry_on RedisRateLimiter::LimitError,
-           wait: :exponentially_longer,
+           wait: :polynomially_longer,
```

Comment:
This is not a change in behavior, just a rename to be more accurate: https://github.com/rails/rails/pull/49292